enhancements/operators/condition-links.md

```diff
@@ -0,0 +1,206 @@
+---
+title: operator-condition-links
+authors:
+  - "@wking"
+reviewers:
+  - "@abhinavdahiya"
+  - "@LalatenduMohanty"
+approvers:
+  - "@deads2k"
+  - "@sdodson"
+creation-date: 2020-04-24
+last-updated: 2020-04-30
+status: implementable
+---
+
+# Operator Condition Links
+
+When an operator condition is in a surprising state (`Available=False`, `Degraded=True`, etc.), it's not always clear from the short reason and message what the impact or recommended mitigations are.
+By including structured links in the condition, we make it easy for all condition consumers to point to external documentation, where there is more space for detailed discussion.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+Are there restrictions on URI targets?
+Do they all need to point at either docs.openshift.com or Knowledgebase articles?
```

Comment:
IMO yes we should have restrict it to couple of places. The URLs should be under control of Red Hat as we do not want customers to go to any unavailable URL. Also we should not have to sen a PR to fix a doc URL because Kubernetes (or any external website) has changed the location of the document. The only challenge with knowledge base articles is that customers without subscription will not be able to access it. Which is bad from OKD point of view.

Reply:
So you would require links to target any RH-controlled site?  Or restrict to a subset of those?  Do you count GitHub repos with RH maintainers as RH-controlled?

Do you have suggestions for supporting OKD?  My fuzzy impression is that Knowledgebase articles can be set public, but we could also require folks to put these in Bugzilla comments or something if I'm misremembering.